book/09-git-and-other-scms/sections/client-tfs.asc

```diff
@@ -1,54 +1,54 @@
+Ang suporta nito sa TFVC na mga tampok ay sobrang gulang, dahil ito ay gumagamit ng Visual Studio na mga assembly para sa mga operasyon sa mga server.
+Ito ay nangangahulugan na kinakailangan mo ng access sa mga assembly na iyon, na nangangahulugang dapat kang mag-install ng isang kamakailang bersyon ng Visual Studio (anumang edisyon mula sa 2010 na bersyon, na nagsasama ng Express mula sa 2012 na bersyon), o ang Visual Studio na SDK.
 
-Git-tf (whose home is at https://gittf.codeplex.com[]) is a Java project, and as such runs on any computer with a Java runtime environment.
-It interfaces with Git repositories through JGit (a JVM implementation of Git), which means it has virtually no limitations in terms of Git functions.
-However, its support for TFVC is limited as compared to git-tfs – it does not support branches, for instance.
+Ang Git-tf (na ang tahanan ay nasa https://gittf.codeplex.com[]) ay isang Java na proyekto, at dahil dito ay tumatakbo sa anumang kompyuter kasama ang isang Java runtime na enviroment.
+Nag-i-interface ito gamit ang Git na mga repostoryo sa pamamagitan ng JGit (isang JVM na pagpapatupad ng Git), na nangangahulugan na ito ay may birtwal na walang mga limitasyon sa termino ng mga function ng Git.
+Samantala, ang suporta nito para sa TFVC ay limitado kaysa sa kung ano ang mayroon ng git-tfs – hindi ito sumusuporta ng mga branch, halimbawa.
 
-So each tool has pros and cons, and there are plenty of situations that favor one over the other.
-We'll cover the basic usage of both of them in this book.
+Kaya bawat kasangkapan ay mayroong mga kalamangan at kahinaan, at mayroong maraming mga sitwasyon na pabor sa isa kaysa sa isa pa.
+Sasakupin natin ang mga paunang paggamit nitong dalawa sa librong ito.
 
-[NOTE]
+[TANDAAN]
 ====
-You'll need access to a TFVC-based repository to follow along with these instructions.
-These aren't as plentiful in the wild as Git or Subversion repositories, so you may need to create one of your own.
-Codeplex (https://www.codeplex.com[]) or Visual Studio Online (http://www.visualstudio.com[]) are both good choices for this.
+Kakailanganin mo ng access sa isang TFVC na nakabase na repositoryo upang sumunod sa mga tagubiling ito.
+Ang mga ito ay hindi kasing dami sa kagubatan kagaya ng Git o Subversion na mga repositoryo, kaya marahil dapat kang gumawa ng isa para sa iyong sarili.
+Ang Codeplex (https://www.codeplex.com[]) o Visual Studio Online (http://www.visualstudio.com[]) ay parehong magandang pagpipilian para rito.
 ====
 
 
-===== Getting Started: `git-tf`
+===== Pagsisimula: `git-tf`
 
-The first thing you do, just as with any Git project, is clone.
-Here's what that looks like with `git-tf`:
+Ang unang bagay na gagawin mo, katulad ng anumang Git na proyekt, ay mag-clone.
```

Comment:
proyekt to proyekto

Reply:
one last update @josteinfrancois and we can merge this.

Reply:
Changing it now